src/coprocessor/statistics/analyze.rs

```diff
@@ -272,9 +278,11 @@ impl<S: Snapshot> RequestHandler for AnalyzeContext<S> {
                 let col_req = self.req.take_col_req();
                 let storage = self.storage.take().unwrap();
                 let ranges = std::mem::take(&mut self.ranges);
+
                 let mut builder =
                     RowSampleBuilder::new(col_req, storage, ranges, self.quota_limiter.clone())?;
-                let res = AnalyzeContext::handle_full_sampling(&mut builder).await;
+
+                let res = AnalyzeContext::handle_full_sampling(&mut builder, self.is_auto).await;
```

Comment:
can we pass it to `RowSampleBuilder` instead?

Reply:
> can we pass it to `RowSampleBuilder` instead?

Changed.